template.tpl

```diff
@@ -1739,15 +1787,20 @@ const onsuccess = () => {
         normalizeTable(data.trackFormsLinksProperties, 'name', 'value') || null
       );
       break;
-      
+
+    case 'start_session_recording':
+    case 'stop_session_recording':
```

Comment:
it looks like these cases are generally in alphabetical order, so maybe move before `time_event`. alternately, maybe it makes sense to make one single block for all the methods that don't take any arguments?

Reply:
I think condensing the zero arity cases makes sense. Will clean this up in addition to the Object.assign recommendation. 👍